recipes/ignition-cmake/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.10.2)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(ignition-cmake2)
+
+include(IgnUtils)
```

You'll also want to add a dummy `test_package/example.cpp` containing e.g.:
```c++
#Include <iostream>

int main() {
    std::cout << "Hello world!\n";
    return 0;
}
```

```suggestion
include(IgnUtils)

ign_configure_build()
ign_add_executable(${PROJECT_NAME} test_package.cpp)
```